src/python/tests/test_repair_mlss_tags.py

```diff
@@ -0,0 +1,138 @@
+            database: Unit test database (fixture).
+
+        """
+        # Use type(self) instead of self as a workaround to @classmethod decorator (unsupported by pytest and
+        # required when scope is set to "class" <https://github.com/pytest-dev/pytest/issues/3778>)
+        type(self).dbc = database.dbc
+
+    @pytest.mark.parametrize(
+        "mlss_tag, alt_queries, exp_stdout, exp_tag_value, expectation",
+        [
+            ('', [], set(['No repair option has been selected: Nothing to do']), {}, does_not_raise()),
+            ('max_align', [], set(['']),
+             {1: 161, 2: 163, 3: 139, 4: 52068, 5: 2452, 6: 37683, 7: 13002, 8: 825240, 9: 996143, 10: 9708},
+             does_not_raise()),
+            ('msa_mlss_id', [], set(['']), {5: 4, 7: 6, 9: 8, 50001: 4, 50002: 6, 50003: 8},
+             does_not_raise()),
+            (
+                'max_align',
+                [
+                    "UPDATE method_link_species_set_tag SET value = 1 "
+                        "WHERE method_link_species_set_id = 2 AND tag = 'max_align';",
+                    "DELETE FROM method_link_species_set_tag "
+                        "WHERE method_link_species_set_id = 6 AND tag = 'max_align';",
+                    "INSERT INTO method_link_species_set_tag VALUES (404, 'max_align', 1);"
+                ],
+                set([
+                    "Repaired MLSS tag 'max_align' for MLSS id '2'",
+                    "Added missing MLSS tag 'max_align' for MLSS id '6'",
+                    "Deleted unexpected MLSS tag 'max_align' for MLSS id '404'"
+                ]),
+                {1: 161, 2: 163, 3: 139, 4: 52068, 5: 2452, 6: 37683, 7: 13002, 8: 825240, 9: 996143,
+                 10: 9708},
+                does_not_raise()
+            ),
+            (
+                'msa_mlss_id',
+                [
+                    "UPDATE method_link_species_set_tag SET value = 1 "
+                        "WHERE method_link_species_set_id = 5 AND tag = 'msa_mlss_id';",
+                    "DELETE FROM method_link_species_set_tag "
+                        "WHERE method_link_species_set_id = 50001 AND tag = 'msa_mlss_id';",
+                    "INSERT INTO method_link_species_set_tag VALUES (404, 'msa_mlss_id', 1);"
+                ],
+                set([
+                    "Repaired MLSS tag 'msa_mlss_id' for MLSS id '5'",
+                    "Added missing MLSS tag 'msa_mlss_id' for MLSS id '50001'",
+                    "Deleted unexpected MLSS tag 'msa_mlss_id' for MLSS id '404'"
+                ]),
+                {5: 4, 7: 6, 9: 8, 50001: 4, 50002: 6, 50003: 8},
+                does_not_raise()
+            ),
+        ]
+    )
+    def test_repair_mlss_tag(self, mlss_tag: str, alt_queries: List[str], exp_stdout: Set[str],
+                             exp_tag_value: Dict[int, int], expectation: ContextManager) -> None:
+        """Tests `repair_mlss_tags.py` script when all MLSS tags are correct.
```

Comment:
Not sure I understand the description - what are you testing if they are all correct? Or did you want to say that you start with correct ones and within the test first make them incorrect and then test `repair_mlss_tags.py` on that data?

Reply:
This is actually the old description of the method before I refactored everything :facepalm: Will fix it now.

Reply:
```suggestion
        """Tests `repair_mlss_tags.py` script, including its output.
```